simulator/src/plotArea.js

```
@@ -432,12 +432,26 @@ export function setupTimingListeners() {
     $('.timing-diagram-calibrate').on('click', () => {
         plotArea.calibrate();
     })
-    $('.timing-diagram-resume').on('click', () => {
+    const ResumePause = document.querySelector('.ResumePause');
```

ESLint

[error] Missing semicolon.

(semi)